Products/ZenUI3/browser/resources/js/zenoss/monitoringTemplates/datasource.js

```diff
@@ -10,34 +10,18 @@
 
 (function(){
 
```

Comment:
refactored DataSource tree grid component - now each DataSource grid will have own floating windows and they will work in scope of this DataSource.
Also use "itemId" instead of "id" in components.